docs/keycodes.md

```diff
@@ -298,7 +298,7 @@ This is a reference only. Each group of keys links to the page documenting their
 |`LM(layer, mod)`|Momentarily turn on `layer` (like MO) with `mod` active as well.  Where `mod` is a mods_bit.  Mods can be viewed [here](https://docs.qmk.fm/#/feature_advanced_keycodes?id=mod-tap).  Example Implementation: `LM(LAYER_1, MOD_LALT)`|
 |`LT(layer, kc)` |Turn on `layer` when held, `kc` when tapped                                       |
 |`TG(layer)`     |Toggle `layer` on or off                                                          |
-|`TO(layer)`     |Turn on `layer` when pressed                                                      |
+|`TO(layer)`     |Replace active layer stack with only `layer` and the default layer                |
```

Comment:
Honestly, I'm not sure I'm happy with this wording.  It feels like it's on the technical side.  Which could be a problem.


Maybe this would be better? ,
```suggestion
|`TO(layer)`     |Turns on `layer` and turns off all other layers, except the default layer |
```